keyboards/draytronics/elise/readme.md

```diff
@@ -15,17 +15,16 @@ Entering DFU mode (to allow flashing):
  - Pressing the reset button on the back of the board when it is plugged in will enter DFU.
  - Holding down the ESC key whilst plugging in the keyboard will enter DFU.
  - If you have one of the provided keymaps flashed, then pressing FN-Tab will enter DFU.
-    
+
 Make example for this keyboard (after setting up your build environment):
 
-    make draytronics/elise:default
+    qmk compile -kb draytronics/elise -km default
```

Comment:
Please revert this as it is preferred to have the `make` example in keyboard readme's.